natiami/assignment1/Q2.cpp

```diff
@@ -0,0 +1,61 @@
+#include<iostream>
+#include<string>
+#include<vector>
+#include<fstream>
+#include <algorithm>
+#include <array> 
+
+using namespace std;
+
+template<typename T>
+class List{
+public:
+	T key;
+	List<T>* next;
+
+	List(T k){ key = k; }
+};
+
+template<typename T>
+List<T>* findKthElement(List<T>* list, int k){

```

Comment:
Very Good,
The function is separate from I/O code, and is short and concise :)

Extra credits:
What happens if the input is an empty list?
What happens if `k` is zero? `k` is negative? `k` is too large?